src/awx_plugins/interfaces/_temporary_private_container_api.py

```diff
@@ -18,6 +18,7 @@
 def get_incontainer_path(
         path: os.PathLike[str] | str,
         private_data_dir: os.PathLike[str] | str,
+        container_root: os.PathLike[str] | str = '',
```

Comment:
There's a discussion to be had about this API. Should the new argument allowed to be positional? As the number of arguments grow, it's usually best to disallow positional args because it hurts readability and ease of perception on the calling side.

Reply:
is now keyword only.